cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/CopyResourcesTask.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.packerina.task;
+
+import org.ballerinalang.packerina.buildcontext.BuildContext;
+import org.ballerinalang.packerina.buildcontext.BuildContextField;
+import org.wso2.ballerinalang.compiler.tree.BLangPackage;
+import org.wso2.ballerinalang.compiler.util.ProjectDirConstants;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.Collections;
+import java.util.List;
+
+import static org.ballerinalang.tool.LauncherUtils.createLauncherException;
+
+/**
+ * Copy resources to module jar.
+ *
+ * @since 1.1.2
+ */
+public class CopyResourcesTask implements Task {
+
+    @Override
+    public void execute(BuildContext buildContext) {
+        Path sourceRoot = buildContext.get(BuildContextField.SOURCE_ROOT);
+        List<BLangPackage> moduleBirMap = buildContext.getModules();
+        for (BLangPackage module : moduleBirMap) {
+            copyResourcesToJar(buildContext, sourceRoot, module);
+        }
```

Comment:
What about transitive dependencies ?

Reply:
We are copying to module jar hence those should be automatically added in executable.
We can't add resources in executable jar since for 'ballerina run' we are using thin jars for class path instead of creating executable.